ArduCopter/mode_guided.cpp

```diff
@@ -21,11 +21,14 @@ struct {
     float roll_cd;
     float pitch_cd;
     float yaw_cd;
+    float roll_rate_cds;
+    float pitch_rate_cds;
     float yaw_rate_cds;
     float climb_rate_cms;   // climb rate in cms.  Used if use_thrust is false
     float thrust;           // thrust from -1 to 1.  Used if use_thrust is true
     bool use_yaw_rate;
     bool use_thrust;
+    bool use_body_rate;
```

Comment:
we should probably rename this to "use_roll_pitch_rates"

Reply:
This is refactored in bfbca89.